### ISSUE TITLE
AssociationShutdownException - do not log it as an error

### ISSUE THREAD

**Issue by Aaronontheweb** [akkadotnet/akka.net#1567]
`AssociationShutdownException` is raised when a transport is explicitly shutdown via a message. This should be logged as an "info" message and not necessarily a giant red, scary, error message. Going to change this so it stops bloating up the logs on our Akka.Remote benchmarks and tests.

**Comment by tstojecki** [comment]
It looks like this particular exception is being logged as Debug in Akka.Remote\EndpointManager.cs
However there are other exceptions in there that are logged as Error, Warning, for example  InvalidAssociation (Warning) or HopelessAssociation  (Error)

https://github.com/akkadotnet/akka.net/blob/d14ad1ef01adbf26e0a53ae41de922273924dcf3/src/core/Akka.Remote/EndpointManager.cs#L369-L437

I am not sure if there is room for consolidation on this. I wouldn't necessarily say that should log all these warnings as Debug. At the same time, I agree it can be unnecessarily disturbing depending on a situation.

Thoughts?

**Comment by tstojecki** [comment]
hi @Aaronontheweb 
I feel like we need to progress this stuff and I can take on that with a little bit of guidance. I have set up your sample remote project locally so that I can test the scenarios fairly easily. Right now the association messages are all over the place, you will see in the screenshot I have attached that it outputs a warning and an error message all at once. This stuff is a pretty hard sell to a QA team.

![image](https://cloud.githubusercontent.com/assets/1306272/12558090/26190b54-c38e-11e5-9a94-d9eca5143b74.png)
